tests/bdd/features/volume/topology/test_node_topology.py

```diff
@@ -0,0 +1,519 @@
+
+    # Check for a pools
+    pools = ApiClient.pools_api().get_pools()
+    assert len(pools) == 12
+    # Check for a nodes
+    nodes = ApiClient.nodes_api().get_nodes()
+    assert len(nodes) == 4
+    node1 = ApiClient.nodes_api().get_node(NODE_1_NAME)
+    assert node1["spec"]["labels"] == {"alpha1": "beta1"}
+    node2 = ApiClient.nodes_api().get_node(NODE_2_NAME)
+    assert node2["spec"]["labels"] == {"alpha2": "beta2"}
+    node3 = ApiClient.nodes_api().get_node(NODE_3_NAME)
+    assert node3["spec"]["labels"] == {"alpha3": "beta3"}
+    node4 = ApiClient.nodes_api().get_node(NODE_4_NAME)
+    assert node4["spec"]["labels"] == {"alpha1": "beta1", "alpha2": "beta2", "alpha3": "beta3"}
+    
+
+
+@given(parsers.parse('a request for a {replica} replica volume with nodeAffinityTopologyLabel as {node_affinity_topology_label} and node topology inclusion as {volume_node_topology_inclusion_label}'))
+def a_request_for_a_replica_replica_volume_with_nodeaffinitytopologylabel_as_node_affinity_topology_label_and_node_topology_inclusion_as_volume_node_topology_inclusion_label(create_request, replica, node_affinity_topology_label, volume_node_topology_inclusion_label):
+    """a request for a <replica> replica volume with nodeAffinityTopologyLabel as <node_affinity_topology_label> and node topology inclusion as <volume_node_topology_inclusion_label>."""
+    if node_affinity_topology_label == "True":
+        request = create_volume_body(replica, volume_node_topology_inclusion_label)
+        create_request[CREATE_REQUEST_KEY] = request
+
+@given(parsers.parse('a request for a {replica} replica volume with nodeHasTopologyKey as {has_topology_key} and node topology inclusion as {volume_node_topology_inclusion_label}'))
+def a_request_for_a_replica_replica_volume_with_nodehastopologykey_as_has_topology_key_and_node_topology_inclusion_as_volume_node_topology_inclusion_label(create_request, replica, has_topology_key, volume_node_topology_inclusion_label):
+    """a request for a <replica> replica volume with nodeHasTopologyKey as <has_topology_key> and node topology inclusion as <volume_node_topology_inclusion_label>."""
+    if has_topology_key == "True":
+        request = create_volume_body(replica, volume_node_topology_inclusion_label)
+        create_request[CREATE_REQUEST_KEY] = request
+
+
+@when(parsers.parse('the desired number of replica of volume i.e. {replica} here; is {expression} number of the nodes having the label {volume_node_topology_inclusion_label}'))
+def the_desired_number_of_replica_of_volume_ie_replica_here_is_expression_number_of_the_nodes_having_the_label_volume_node_topology_inclusion_label(create_request, replica, expression, volume_node_topology_inclusion_label):
+    """the desired number of replica of volume i.e. <replica> here; is <expression> number of the nodes having the label <volume_node_topology_inclusion_label>."""
+    if expression == "less than or equal to":
```

Comment:
I wonder if we should these instead of words?

```suggestion
    if expression == "<=":
```